controllers/volumereplicationgroup_controller.go

```diff
@@ -712,7 +712,7 @@ func (v *VRGInstance) reconcileVRsForDeletion() bool {
 			continue
 		}
 
-		log.Info("Successfully processed VolumeReplication for PersistentVolumeClaim")
+		log.Info("Successfully processed VolumeReplication for PersistentVolumeClaim", "VR instance", v.instance.ObjectMeta.Name, "PVC", pvcNamespacedName)
```

Comment:
Just `v.instance.Name` is enough. It may even resolve the linter length problem :)